### PR TITLE
docs(outputs.cratedb): Correct required SQL table schema

### PR DESCRIPTION
**Pull request by schmidt-software**
add missing column "day" in sql statement in readme

# Required for all PRs

- [x] Updated associated README.md.
- [ ] Wrote appropriate unit tests.
- [ ] Pull request title or commits are in [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/#summary)

resolves #

### CODE REVIEW

**Comment by powersj** [comment]
@schmidt-software,

Thanks for the PR. I would like to point to documentation in our docs that states the day field is required as a part of this. Thanks again!

**Comment by Hipska** [comment]
> Are you sure the `day` field is **required** for the database? If so, can you please link to a documentation?!

It was already referenced as the part of the primary key and also for the partitioning, so I think it was a typo to not include it in the schema.

> Furthermore, you should use spaces instead of tabs for indentation as reported by the linter...

Second that..

File: plugins/outputs/cratedb/README.md [hunk 1]
```diff
@@ -8,13 +8,14 @@ protocol](https://crate.io/docs/crate/reference/protocols/postgres.html).
...
-  "fields" OBJECT(DYNAMIC),
-  PRIMARY KEY ("timestamp", "hash_id","day")
+CREATE TABLE IF NOT EXISTS my_metrics (
+	"hash_id" LONG INDEX OFF,
+	"timestamp" TIMESTAMP,
+	"name" STRING,
+	"tags" OBJECT(DYNAMIC),
+	"fields" OBJECT(DYNAMIC),
+	"day" TIMESTAMP GENERATED ALWAYS AS date_trunc('day', "timestamp"),
+	PRIMARY KEY ("timestamp", "hash_id","day")
```
Comment (Hipska) [inline plugins/outputs/cratedb/README.md:18]:
```suggestion
  "hash_id" LONG INDEX OFF,
  "timestamp" TIMESTAMP,
  "name" STRING,
  "tags" OBJECT(DYNAMIC),
  "fields" OBJECT(DYNAMIC),
  "day" TIMESTAMP GENERATED ALWAYS AS date_trunc('day', "timestamp"),
  PRIMARY KEY ("timestamp", "hash_id","day")
```

**Review by Hipska — CHANGES_REQUESTED** [review]

**Comment by Hipska** [comment]
@srebhan could you apply my suggestion? I think we are then OK to merge?

**Review by Hipska — APPROVED** [review]

**Review by srebhan — APPROVED** [review]
Thanks for the corrections and explanation @Hipska and thanks for the original fix @schmidt-software!

**Pull request merged**